### DISCUSSION TITLE
Windows instalation / Linux Instalation

### DISCUSSION THREAD

**Discussion by Skyblockpro1** [AloneLiberty/FlipperNestedRecovery#19] [Q&A]
Hi guys, thanks in advance for any help.

Now I have tried to install the tool on windows, that said this isn't my first rodeo with pip, but i honestly got stuck.
Initially I had issue with the VS code or some C++ compilation. 

Once resolved that there are more VS errors.

Than I tried to install it on Ubuntu, on ubuntu it builds correctly, so progress but then it errors out on "Serial" genuine no idea why. I run it as sudo so it has all the privileges it should need.
If you have any ideas in both situation, I would be verry happy to hear 

thankyou :)

**Comment by AloneLiberty** [comment]
I've updated workflows to build for Python 3.12, should work now. Windows uses custom libraries for build and won't build as is. If you want to compile it yourself, check how it done in GitHub Actions
